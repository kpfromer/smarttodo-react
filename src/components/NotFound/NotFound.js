import React, { PureComponent } from 'react';

import styles from './NotFound.module.scss';

export default class NotFound extends PureComponent {
  render() {
    return (
      <div className={styles.container}>
        <h1 className={styles.title}>Page Not Found</h1>
      </div>
    );
  }
}
