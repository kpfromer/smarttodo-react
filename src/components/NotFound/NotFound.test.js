import NotFound from "./NotFound";
import { SetupComponent } from "react-component-setup";

const { shallow: setup } = SetupComponent({
  Component: NotFound
});

describe('NotFound', () => {
  it('renders without crashing', () => {
    setup();
  });

  it('renders a not found message', () => {
    const { wrapper } = setup();

    expect(wrapper).toMatchSnapshot();
  });
});