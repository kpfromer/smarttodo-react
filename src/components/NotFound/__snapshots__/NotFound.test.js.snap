// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders a not found message 1`] = `
<div
  className="container"
>
  <h1
    className="title"
  >
    Page Not Found
  </h1>
</div>
`;
